modules/core/include/opencv2/core/dualquaternion.inl.hpp

```diff
@@ -357,14 +356,20 @@ Vec<T, 8> DualQuat<T>::toVec() const
    return Vec<T, 8>(w, x, y, z, w_, x_, y_, z_);
 }
 
+template <typename T>
+Affine3<T> DualQuat<T>::toAffine3() const
+{
+    return Affine3<T>(toMat());
+}
+
 template <typename T>
 Matx<T, 4, 4> DualQuat<T>::toMat() const
```

Comment:
Here (and in `toAffine3`) an `assumeUnit` argument can be added for `getRotation` call

Reply:
OK